docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -653,6 +653,102 @@ exports.handler = async function(params) {
 }
 ----
 
+[[customizing-notification-messages]]
+== Customizing Notification Messages
+
+You can optionally modify the notification message content and formatting using the checkbox below the notification channel selector.
+
+=== Message Syntax
+
+Custom notifications support a limited set of markdown syntax:
+
+* Bold
+* Italic
+* Links
+
+There is partial support for additional markdown syntax, but rendering behavior varies by platform.
```

Comment:
If we're running out of time we can leave like this, but ideally I'd like to see a bit more discussion around this. Reading it feels like there *could* be support for some things I might want to use but there's no way to confirm it.

Reply:
I thought about just including the cue that they will see a warning if using any features with partial support, but provided a little more context than that. If it feels too wordy now, I can cut it down - let me know what you think.